modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+// * rosa-adding-additional-constraints-for-ip-based-aws-role-assumption/rosa-create-an-identity-based-policy.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-create-an-identity-based-policy_{context}"]
+= Create an Identity-based IAM policy
+
+:context: rosa-create-an-identity-based-policy
+
+toc::[]
+
+You can create an Identity-based Identity and Access Management (IAM) policy that denies access to all AWS actions when the request originates from an IP address other than Red Hat provided IPs.
+
+.Prerequisites
+
+* You have access to the AWS Management Console with the permissions required to create and modify IAM policies.
```

Comment:
Consider adding a link to the AWS Management Console in the prerequisites.

Reply:
Added a link to AWS Management Console. Please confirm.